python/lsst/ctrl/events/eventsLib.i

```diff
@@ -61,15 +61,24 @@ Access to the lsst::ctrl::events classes
 
 %shared_ptr(lsst::ctrl::events::EventFormatter)
 
+
 %import "lsst/daf/base/baseLib.i"
 %import "lsst/pex/logging/loggingLib.i"
-%import "lsst/pex/policy/policyLib.i"
 
+%typemap(out) std::vector<std::string > {
```

Comment:
I tried that, and it gave an error about duplication, which I traced to a directive in baseLib.i which did this:
```
%define VectorAddType(type, typeName)
    %template(Vector ## typeName) std::vector<type>;
%enddef
```
and later:
```
VectorAddType(std::string, String)
```

Since that was already being included, I'm not sure why getting a Python list back wasn't working in the first place, and why the code I took from pex_policy was necessary.  It also includes baseLib.i.